src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -676,13 +676,13 @@ int TableDelete_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
   size_t len = 0;
   const char *data_ptr = nullptr;
   data_ptr = RedisModule_StringPtrLen(data, &len);
-  REPLY_AND_RETURN_IF_FALSE(
-      len % kUniqueIDSize == 0,
-      "The deletion data length must be a multiple of the UniqueID size.");
-  size_t ids_to_delete = len / kUniqueIDSize;
+  // The first uint16_t are used to encode the number of ids to delete.
+  size_t ids_to_delete = *reinterpret_cast<const uint16_t *>(data_ptr);
+  size_t id_length = (len - sizeof(uint16_t)) / ids_to_delete;
+  data_ptr += sizeof(uint16_t);
```

Comment:
keep the `REPLY_AND_RETURN_IF_FALSE` and check `(len - sizeof(uint16_t)) % ids_to_delete == 0` here?

Reply:
Added.